src/models.rs

```diff
@@ -589,6 +589,10 @@ impl RepositoryUses<'_> {
             _ => None,
         }
     }
+
+    pub(crate) fn as_template(&self) -> String {
+        format!("{}/{}", self.owner, self.repo)
+    }
```

Comment:
Created this new method as a helper, since I need to get a template from what we have in well-know list.

Reply:
I don't love that we have to re-build the slug here, but that's my fault for building a bad abstraction here 😅 -- I'll try and clean this up as part of the final review.